providers/kube/README.md

```diff
@@ -214,10 +214,11 @@ Should your `$MASTER_IP`, example:
 147.75.70.201
 ```
 
-Install Kubernetes by running the below:
+Install Kubernetes by running below commands:
 
 ```shell
 make layer1-install HOST=$HOST MASTER_IP=$MASTER_IP
+make kube-csi-install HOST=$HOST MASTER_IP=$MASTER_IP
```

Comment:
I remember @gosuri removing this from `layer1-install` definition. @gosuri do you think we need this? If so, let's keep it part of `layer1-install` itself. Or may be we can mark it as `optional`.

Reply:
that's only for providers with CSI - like packet